.flux.yaml

```diff
@@ -32,10 +32,10 @@ spec:
     resources:
       requests:
         cpu: "100m"
-    # hpa:
-    #   enabled: true
-    #   minReplicas: 1
-    #   maxReplicas: 10
-    #   targetCPUUtilizationPercentage: 80
+    hpa:
```

Comment:
why was this commented out before and why is it needed now? And what is it doing?